src/Makefile.am

```diff
@@ -243,6 +244,7 @@ libbitcoin_server_a_SOURCES = \
   net.cpp \
   noui.cpp \
   policy/fees.cpp \
+  policy/policy.cpp \
```

Comment:
need to add this to `CMakeLists.txt` as well

Reply:
done 👍